examples/flexvolume/lvm

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+usage() {
+	echo "Invalid usage. Usage: $0 <init/setup/teardown> <pod namespace> <pod name> <pod UID> <volume name> <json params>"
+}
+
+create() {
+
+	VOLUMEID=$(echo $1 | jq -r '.volumeID')
+	SIZE=$(echo $1 | jq -r '.size')
+	VG=$(echo $1|jq -r '.volumegroup')
+
+	DMDEV="/dev/mapper/${VG}-${VOLUMEID}"
+	if [ -b "${DMDEV}" ]; then
+		echo -n "{\"device\":\"${DMDEV}\"}"
+		exit 0
+	fi
+
+	lvcreate -L${SIZE} -n${VOLUMEID} ${VG} > /dev/null 2>&1
+	if [ $? -ne 0 ]; then
+		echo "Failed to create volume $VOLUMEID using VG ${VG}"
```

Comment:
instead of echo plain text errors, I'd like a well-formed output like the following (similar to ansible)

```json
{
 "ok": true|false
 "message": failure_message
 "device": device_path
}
```